src/Service.Tests/SqlTests/RestApiTests/Patch/MsSqlPatchApiTests.cs

```diff
@@ -190,6 +216,101 @@ public async Task PatchOneViewBadRequestTest()
             await base.PatchOneViewBadRequestTest(expectedErrorMessage);
         }
 
+        /// <summary>
+        /// Test to validate successful execution of a request when a rowversion field is missing from the request body.
+        /// </summary>
+        [TestMethod]
+        public async Task PatchOneWithRowversionFieldMissingFromRequestBody()
+        {
+            // Validate successful execution of a PATCH update when a rowversion field (here 'row_version')
+            // is missing from the request body. A PATCH update request should not try to update any field which is not provided in the request body.
+            string requestBody = @"
+            {
+                ""book_name"": ""Another Awesome Book"",
+                ""copies_sold"": 100,
+                ""last_sold_on"": null
+            }";
+            string expectedLocationHeader = $"id/1";
+
+            await SetupAndRunRestApiTest(
+                    primaryKeyRoute: expectedLocationHeader,
+                    queryString: null,
+                    entityNameOrPath: _entityWithReadOnlyFields,
+                    sqlQuery: GetQuery("PatchOneUpdateWithRowversionFieldMissingFromRequestBody"),
```

Comment:
`PatchOneUpdateWithRowversionFieldMissingFromRequestBody` doesn't validate the value of `row_version`

Reply:
We can't know the value of row_version field beforehand. The best we can do is a NOT NULL check. Will incorporate that.